packages/sources/css/src/components/radio-button/src/index.css

```diff
@@ -0,0 +1,93 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+body {
+  font-family: var(--vtmn-typo_font-family);
+}
+
+p {
+  margin: 8px 0;
+}
```

Comment:
You can set this properties directly to your specific class, not generic types like <p> or <body>